### DISCUSSION TITLE
demos是什么文件

### DISCUSSION THREAD

**Discussion by LM-6** [ant-design/ant-design-mobile#4582] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79618846/146675289-2fe736b3-1f37-4600-9ee7-60a1e04010a0.png)

**Comment by GOWxx** [comment, marked as answer]
`demos` 只是用于 `ant-design-mobile` 文档页面展示用的
`demos` 并不由 `antd-mobile` 库提供
在项目代码里不要使用 `demos` 就好了

**Reply by GOWxx** [reply]
以 `Button` 组件为例
`Button` 组件是由 `antd-mobile` 提供的
![image](https://user-images.githubusercontent.com/12870715/146676662-6911b4e9-d75e-4719-93b4-c03536859349.png)

而 `Demos` 里引出的 `DemoBlock` 只是为了提供这个外面的容器
这样可以在文档中更好的说明这部分代码是为了展示什么内容或特性的
![image](https://user-images.githubusercontent.com/12870715/146676726-0ad8c6d8-23f0-4860-9e8b-725572a76fb5.png)

实际开发的时候不要使用 `demos` 相关的代码哦~
